c2corg_ui/templates/helpers/view.html

```diff
@@ -329,37 +334,40 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
         <span class="glyphicon glyphicon-map-marker"></span>
         <span translate>Associated routes</span><span class="glyphicon glyphicon-menu-down"></span>
       </h3>
-      <div class="associated-documents collapse in" id="associated-routes">
-        <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
-          <app-card app-card-doc="doc"></app-card>
-          % if type == 'routes':
-            <app-delete-association child-doctype="routes" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="${test_association_rights(document)}
-              % if document['type'] == 'o':
-                  && ${associations}.length > 1
-              % endif
-            "></app-delete-association>
+      <div class="collapse in" id="associated-routes">
+        <app-activity-filter activities="${activities}" documents="${associations}"></app-activity-filter>
+        <div class="associated-documents">
```

Comment:
The change in this file is mainly that CSS classes ``collapse in`` have been moved to a new parent container so that the ``app-activity-filter`` directive is shown  inside the associated routes section but is not impacted by the card's flex layout.
Most of the changed lines have actually been reindented.